api/v1alpha1/flowcollector_types.go

```diff
@@ -38,9 +42,18 @@ type FlowCollectorSpec struct {
 	// FlowlogsPipeline contains settings related to the flowlogs-pipeline component
 	FlowlogsPipeline FlowCollectorFLP `json:"flowlogsPipeline,omitempty"`
 
-	// Loki contains settings related to the loki client
+	// Loki contains settings related to the loki configuration
 	Loki FlowCollectorLoki `json:"loki,omitempty"`
 
+	// Kafka contains settings related to the kafka configuration
+	Kafka FlowCollectorKafka `json:"kafka,omitempty"`
+
+	// Grafana contains settings related to the grafana configuration
+	Grafana FlowCollectorGrafana `json:"grafana,omitempty"`
+
+	// Prometheus contains settings related to the prometheus configuration
+	Prometheus FlowCollectorPrometheus `json:"prometheus,omitempty"`
```

Comment:
For all these three, we should also mention that providing the configs will trigger installing the operators, and that it's experimental.